hraven-core/src/main/java/com/twitter/hraven/rest/ObjectMapperProvider.java

```diff
@@ -66,10 +69,13 @@ public ObjectMapper getContext(Class<?> type) {
   public static ObjectMapper createCustomMapper() {
     ObjectMapper result = new ObjectMapper();
     result.configure(Feature.INDENT_OUTPUT, true);
-    SimpleModule module = new SimpleModule("hRavenModule", new Version(0, 4, 0, null));
+    SimpleModule module = new SimpleModule("hRavenModule", new Version(0, 4, 0,
```

Comment:
It looks like creating a simple hRavenModule is repeated in a few other places. It might be better to extract it into a method and call it instead.